torch/ao/quantization/observer.py

```diff
@@ -1316,6 +1316,8 @@ class PlaceholderObserver(ObserverBase):
     Args:
         dtype: dtype argument to the `quantize` node needed to implement the
                reference model spec.
+        quant_min: minimum value in quantized domain (TODO: align behavior with other observers)
```

Comment:
what does the TODO mean?  Ideally if it's something important it would be done in the same PR, since observers are widely used with BC guarantees.

Reply:
it means removing this one: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/observer.py#L240-L241 it will be a bigger change and need to handle BC etc. we need to have a rule to say when quant_min/quant_max should be specified I think

Reply:
I'll land for now to unblock the work, I think we will need to fix the tech debts a bit later..